src/components/ProjectListTopBar.tsx

```diff
@@ -1,21 +1,35 @@
 import {
   Heading,
-  Button,
   Flex,
   Spacer,
   HStack,
   IconButton,
+  Menu,
+  MenuItem,
+  MenuButton,
+  MenuList,
+  Avatar
 } from "@chakra-ui/react";
 import { useNavigate } from "react-router-dom";
 import { IoMdAdd } from "react-icons/io";
-import { MaybeUser } from "../contexts/ApiContext";
+import { ApiContext, MaybeUser } from "../contexts/ApiContext";
+import { useContext, useState } from "react";
 
 interface ProjectListTopBarProps {
   onLogout: (token: MaybeUser) => void;
 }
 
 export function ProjectListTopBar({ onLogout }: ProjectListTopBarProps) {
+  const [username, setUsername] = useState("Username Lastname");
+
   const navigate = useNavigate();
+  const client = useContext(ApiContext).client;
+
+  // Get the username to set the profile avatar
+  client.GET("/users/me").then((res: any) => {
+    const username = res.data.username;
+    setUsername(username);
+  });
```

Comment:
This would probably be better in a `useEffect` since otherwise this request gets made every time the component rerenders

Reply:
Fixed! Let me know if I set it up properly.